build.gradle

```diff
@@ -38,6 +38,8 @@ dependencies {
     implementation  group: 'org.apache.ant', name: 'ant', version: '1.10.12'
     implementation  group: 'org.apache.commons', name: 'commons-csv', version: '1.9.0'
     implementation  group: 'org.fusesource.jansi', name: 'jansi', version: '2.4.0'
+    implementation group: 'com.fasterxml.jackson.datatype', name: 'jackson-datatype-jsr310', version: '2.17.0'
+    implementation group: 'com.fasterxml.jackson.dataformat', name: 'jackson-dataformat-yaml', version: '2.17.0'
```

Comment:
Since we are not using any datetimes in our configuration files for now, I believe the first `datatype` dependency is not needed, I will remove it from the list of dependencies.